assets/lang/sklmessages_pl_PL.properties

```diff
@@ -82,6 +84,8 @@ editor.compatibility.tooltip=Skiny nie dzia\u0142aj\u0105 w tym trybie, ale niek
 
 settings.title=Ustawienia launchera
 settings.lang=J\u0119zyk
+settings.lang.help=Widzisz jakie\u015b b\u0142\u0119dy w t\u0142umaczeniu?\u0020
```

Comment:
Is this whitespace character (\u0020) really necessary? You added it also in #34 and @skmedix rejected it without giving a reason, so I don't know it should remain or not.